package.json

```diff
@@ -14,19 +15,10 @@
     "ios",
     "android"
   ],
-  "author": {
-    "name": "Ladislav Martincik",
-    "email": "ladislav.martincik@gmail.com",
-    "url": "https://github.com/martincik"
-  },
   "repository": {
     "type": "git",
-    "url": "https://github.com/slowpath/react-native-hockeyapp"
-  },
-  "bugs": {
-    "url": "https://github.com/slowpath/react-native-hockeyapp/issues"
+    "url": "https://github.com/peartherapeutics/react-native-hockeyapp"
```

Comment:
If you open https://github.com/peartherapeutics in an incognito tab you'll see that all of our forks of public repos also appear to be public. Never noticed this before but there it is.